modules/authlinkedin/lib/Auth/Source/LinkedIn.php

```diff
@@ -46,6 +47,12 @@ public function __construct($info, $config) {
 			throw new Exception('LinkedIn authentication source is not properly configured: missing [secret]');
 
 		$this->secret = $config['secret'];
+
+		if (array_key_exists('attributes', $config))
+			$this->attributes = $config['attributes'];
+		else
+			// Default values if the attributes are not set in config (ref https://developer.linkedin.com/docs/fields)
+			$this->attributes = 'id,first-name,last-name,headline,summary,specialties,picture-url,email-address';
 	}
```

Comment:
I just have one request if you fix the entire file to follow PSR-2, and it's that you do that in separate commits, so that one commit is changing the file to be PSR-2 compliant, and the other is adding the functionality that you want to merge. That way we keep a clean history and it's easier to parse the changes and evaluate the PR.

Thanks a lot for your work @tauceti2!